src/engraving/libmscore/rendermidi.cpp

```diff
@@ -1894,7 +1894,9 @@ std::vector<OrnamentExcursion> excursions = {
     { SymId::ornamentDownMordent,         any, _16th, { 1, 1, 1, 0 }, { 1, 0 },    true,  true, { -1, 0 } },// p136 Cadence Appuyee + mordent [1] [2]
     { SymId::ornamentPrallUp,             any, _16th, { 1, 0 }, { 1, 0 },        true,  true, { -1, 0 } },// p136 Double Cadence [1]
     { SymId::ornamentPrallDown,           any, _16th, { 1, 0 }, { 1, 0 },        true,  true, { -1, 0, 0, 0 } },// p144 ex 153 [1]
-    { SymId::ornamentPrecompSlide,        any, _32nd, {},    { 0 },          false, true, {} }
+    { SymId::ornamentPrecompSlide,        any, _32nd, {},    { 0 },          false, true, {} },
+    { SymId::ornamentTremblementCouperin, any, _32nd, { 1, 1 }, { 0, 1 },        true, true, { 0, 0 } },
+    { SymId::ornamentPinceCouperin,       any, _32nd, { 0 },    { 0, -1 },       true, true, { 0, 0 } }
```

Comment:
For playback, they would need to be handled additionally in `src/engraving/playback/metaparsers/internal/symbolsmetaparser.cpp`.

Reply:
Hmm, now I'm connfused...

Reply:
> only `ornamentTremblementCouperin` is missing

it is also there, line 308
with `mpe::ArticulationType::Tremblement`

Reply:
Oh, I see!